src/Akka.Persistence.SqlServer/sql-server.conf

```diff
@@ -27,6 +27,8 @@
 
 			# metadata table
 			metadata-table-name = Metadata
+
+			sequential-access = off
```

Comment:
Why leave it to `off` by default?

Reply:
@Aaronontheweb wasnt sure exactly what it was for without reading through the common library code too so thought that would be best can easily make it on by default if you want?

Reply:
@Aaronontheweb If I'm right this is aligned to what is the default on sql common in core akka.net repository - also this behavior is backwards compatible.

Reply:
@Horusiath it is backwards compatible either way. The only breaking change was the constructor overload itself, which this PR fixes.

Turning on sequential access by default will significantly reduce the overhead for loading large snapshots as it turns treatment of those byte or character array fields into streams. You can see the full details here: https://msdn.microsoft.com/en-us/library/system.data.commandbehavior(v=vs.110).aspx

I'd turn it on by default - can't see the harm in doing it.

Reply:
If you really want to be sure, do a backwards compat test in the specs somewhere. Do a normal write from a snapshot store followed by two reads from different configurations. If you get the same data, just enable sequential access.